watertap/unit_models/tests/test_reverse_osmosis_1D.py

```diff
@@ -881,6 +904,259 @@ def testReverseOsmosis_cp_calculated_kf_calculated(self):
             m.fs.unit.mixed_permeate[0].flow_mass_phase_comp["Liq", "NaCl"]
         )
 
+    @pytest.mark.component
+    def testRO_Pdrop_calculation(self):
+        m = ConcreteModel()
```

Comment:
A little bit unsure why this test is added in this PR? Don't we include it for other tests or anything changed?

Reply:
Good catch - I  added it in this PR because I thought it was missing. 0D RO and 0D OARO both have a test called Pdrop_calculation that makes sure the system works when PressureChangeType is set to calculated, but `def RO_frame` actually covers that in this file.

Reply:
Yeah, so no need for this test, other things look good to me!